TechnicalProductSummary.rst

```diff
@@ -19,3 +19,24 @@ Starting notes:
 
 Hack-a-thon content:
 --------------------
+When you're working on creating a product summary for a technical product, your first step is to brainstorm your plan. 
```

Comment:
This needs an introduction, as suggested by the "Starting notes".  For example:

Check with Marketing and Sales to see if a technical product summary already exists. If there isn't a summary, you should create one.